client/belga/index.tsx

```diff
@@ -8,6 +8,7 @@ import planningExtension from 'superdesk-planning-extension';
 import markForUserExtension from 'superdesk-core/scripts/extensions/markForUser/dist/src/extension';
 import datetimeFieldExtension from 'superdesk-core/scripts/extensions/datetimeField/dist/src/extension';
 import belgaCoverageExtension from '../extensions/belgaCoverage/dist/index';
+import updateArticleExtension from '../extensions/updateArticle/dist/src/extension';
```

Comment:
Try naming extensions according to features they implement instead of which extension points they use. This one should be probably called `updateArticleOnContentProfileChange` or similar.